README.md

```
@@ -450,43 +432,43 @@
     - 해당 사용자가 부모인 경우, **준비완료! 버튼**을 클릭하면 부모 랜딩 페이지(3.2.4)로 넘어간다.
     - 해당 사용자가 자녀인 경우, **준비완료! 버튼을** 클릭하면 메인 페이지(3.2.6)로 넘어간다.
 
-### **3.2.3 `로그인 페이지`**
+### **3.2.3. 로그인 페이지**
 
 - 비밀번호 입력
     - 비밀번호 입력 필드에 비밀번호를 입력하고 **로그인 버튼**을 클릭하면, 이메일과 비밀번호가 동일한 사용자의 정보인지 확인한다.
     - 정보가 일치하고 해당 사용자가 부모인 경우, 부모 랜딩 페이지(3.2.4)로 이동한다.
     - 정보가 일치하고 해당 사용자가 자녀인 경우, 메인 페이지(3.2.5)로 이동한다.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)